src/db/schema.ts

```diff
@@ -69,12 +69,20 @@ export const eventType = pgEnum("event_type", [
 ]);
 export const memberRank = pgEnum("member_rank", ["aspirante", "bulto", "tuno"]);
 export const userRole = pgEnum("user_role", ["admin", "editor", "viewer"]);
+export const authProvider = pgEnum("auth_provider", [
+  "local",
+  "google",
+  "clerk",
+]);
 
 export const users = pgTable(
   "users",
   {
     id: uuid().defaultRandom().primaryKey().notNull(),
-    clerkUserId: text("clerk_user_id"),
+    email: varchar("email", { length: 255 }).notNull().unique(),
+    passwordHash: text("password_hash").notNull(),
+    provider: authProvider().default("local"),
+    providerId: text("provider_id"),
```

Comment:
this is probably fine for this use case. In some cases you might want to have multiple auth providers for one user but this is probably considered out of scope.

Reply:
I also saw that password hash is not null, can a user not sign up with just google?